tools/gn

```diff
@@ -189,20 +234,14 @@ def to_gn_args(args):
       # The GN arg is not available in the windows toolchain.
       gn_args['enable_lto'] = enable_lto
 
-    if args.target_os == 'android':
-        gn_args['target_os'] = 'android'
-    elif args.target_os == 'ios':
-        gn_args['target_os'] = 'ios'
-        gn_args['use_ios_simulator'] = args.simulator
+    if args.target_os:
+      gn_args['target_os'] = get_target_os(args.target_os)
```

Comment:
Right now, it looks like this could be simplified down to `gn_args['target_os'] = args.target_os`. Will `get_target_os` be doing something more in subsequent changes?

Reply:
Just looked at the diffs for my followup patch and the only diff I have is that I raise an exception if the target_os isn't one in our list. We can kill it off.